packages/suite-desktop-core/e2e/playwright.config.ts

```diff
@@ -27,19 +27,25 @@ const config: PlaywrightTestConfig = {
     testDir: 'tests',
     workers: 1, // to disable parallelism between test files
     use: {
+        viewport: { width: 1280, height: 720 },
         headless: process.env.HEADLESS === 'true',
         trace: 'on',
         video: 'on',
         screenshot: 'on',
         testIdAttribute: 'data-testid',
-        actionTimeout: 30000,
```

Comment:
I had some issue with traces and fails. Once I removed this forced timeout, it got fixed. Every action has its default timeout. So I say lets leave it to carefully chose defaults by the playwright devs.